pom.xml

```diff
@@ -39,10 +39,19 @@
         </dependency>
         
         <!-- LOGGING -->
+        <!-- http://mvnrepository.com/artifact/org.slf4j/slf4j-simple -->
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.21</version>
```

Comment:
Replace the version with a property in order to avoid misalignment with slf4j-simple in test scope.